python/GafferImageTest/OpenImageIOReaderTest.py

```diff
@@ -348,6 +348,9 @@ def testMissingFrameMode( self ) :
 		# set to a missing frame
 		context.setFrame( 2 )
 
+		# FormatPlug to query default format from context
```

Comment:
You shouldn't need to create an instance of FormatPlug for this, because `getDefaultFormat()` is a static method.